2022q2/pf.adoc

```diff
@@ -0,0 +1,37 @@
+=== pf status update
```

Comment:
That markup looks wrong. It should be == for the section header and === for the other 3, not the opposite.

Reply:
pauamma is right in saying that the markup is wrong, but the needed fix is different:
- leave the section header with === as it is;
- use ==== for subsections.

See also https://raw.githubusercontent.com/freebsd/freebsd-quarterly/main/2022q1/freebsd-foundation.adoc for an example.